src/main/java/org/breedinginsight/services/ProgramService.java

```diff
@@ -223,4 +229,14 @@ public ProgramBrAPIEndpoints getBrapiEndpoints(UUID programId) throws DoesNotExi
         return dao.getProgramBrAPIEndpoints(programId);
     }
 
+    private boolean programNameInUse(String name) {
+        List<ProgramEntity> existingPrograms = dao.fetchByName(name);
```

Comment:
Is this check case-insensitive? Do we want a case insensitive match?

Reply:
> Is this check case-insensitive? Do we want a case insensitive match?

I think it's case-sensitive. I'll ask in the channel about a case insensitive match.

Reply:
> > Is this check case-insensitive? Do we want a case insensitive match?
> 
> I think it's case-sensitive. I'll ask in the channel about a case insensitive match.

We're going with case insensitive so I modified it accordingly.